sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -359,6 +375,10 @@ void tryGetValuesFromSystem() {
         String numberOfOperationsValue = StringUtils.defaultString(
                 Strings.emptyToNull(System.getenv().get("NUMBER_OF_OPERATIONS")), Integer.toString(numberOfOperations));
         numberOfOperations = Integer.parseInt(numberOfOperationsValue);
+
+        String  throughputForMultiClientValue = StringUtils.defaultString(
```

Comment:
code style: you have multiple spaces `String  throughputForMultiClientValue` -> `String throughputForMultiClientValue`